lib/Model/Importer/Binary/MipsELFImporter.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+using namespace llvm;
+
+template <typename T, bool HasAddend>
+class MipsELFImporter {
+  llvm::Optional<uint64_t> SymbolsCount;
+  llvm::Optional<uint64_t> MIPSFirstGotSymbol;
+  llvm::Optional<uint64_t> MIPSLocalGotEntries;
```

Comment:
We're deprecating `llvm::Optional` in favor of `std::optional`.

Reply:
I will put a TODO for this and resolve it as a separate commit since it will take some changes in the methods def.

Reply:
Ping